complete/src/main/resources/application.properties

```diff
@@ -1,5 +1 @@
-spring.h2.console.enabled=true
-server.port=8080
-aws.secretsmanager.secretName=Prod
-aws.secretsmanager.region=ap-southeast-1
-cognito.jkws.url=https://cognito-idp.ap-southeast-1.amazonaws.com/ap-southeast-1_iQdl5AVrA/.well-known/jwks.json
\ No newline at end of file
+spring.profiles.active=local
```

Comment:
If it's possible to keep the common properties that apply to all profiles in the `application.properties` file, and only the changing properties in the other properties file, it would be nice. So, can we keep `server.port` and `cognito.jkws.url` in the  `application.properties` and remove it from the other profile specific properties file?

Lets set `spring.h2.console.enabled=true` only for local.